lib/authorizer/oauth1.js

```diff
@@ -246,6 +246,29 @@ module.exports = {
         done(null, true);
     },
 
+    /**
+     * Updates the encoding for query parameters in request to RFC-3986.
+     * decodeFrom here decodes breaks the query string down and does
+     * percent decoding on parameters too.
+     *
+     * @param {Request} request - request to update query parameters
+     * @param {Object} url - Node.js like url object
+     */
+    updateQueryParamEncoding: function (request, url) {
```

Comment:
We don't want to export this utility function from this module.
This should be a private function like `getOAuth1BaseUrl`.